src/application/tasks/ventilation.c

```diff
@@ -6,13 +6,11 @@
 
 #include <stdbool.h>
 #include <stdint.h>
-
-#include "platform/system_info.h"
-#include "platform/sensor.h"
-#include "platform/valve.h"
```

Comment:
<> are used for C libraries or system libraries

"" are used for our own files